skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -49,12 +49,12 @@ public ResponseEntity<GenericResponse<EndorsementDTO>> getEndorsementById(@PathV
     }
 
     @PostMapping(value="")
-    public ResponseEntity<?> postEndorsement(@RequestBody @Valid EndorsementDRO endorsementDRO) {
+    public ResponseEntity<GenericResponse<EndorsementModel>> postEndorsement(@RequestBody @Valid EndorsementDRO endorsementDRO) {
```

Comment:
why are we returning a Model, I feel DTO will be better, as we have only specified fields.